### PR TITLE
fix(alearm246): set domain attribute specifically to subdomain

### PR DESCRIPTION
**Pull request by alearm246**
I set the domain attribute for the HTTP only cookie specifically to the staging.devlaunchers.com subdomain only on the staging server to only let the cookie be shared with that subdomain. This will hopefully fix signup up issue on the staging server.

### CODE REVIEW

File: api/custom/controllers/custom.js [hunk 1]
```diff
@@ -10,6 +10,13 @@ module.exports = {
         maxAge: 1000 * 60 * 60 * 24 * 14, // 14 Day Age,
         domain: 'localhost'
       });
+    } else if(process.env.NODE_ENV === 'staging') {
```
Comment (chungthuang) [inline api/custom/controllers/custom.js:13]: Instead of hard coding the domain, we can make it configurable and simplify `if, else if, else` to `if else`.

Comment (alearm246) [reply api/custom/controllers/custom.js:13]: Oh you mean by making domain into an environment variable and setting it to staging.devlaunchers.com in staging?

Comment (chungthuang) [reply api/custom/controllers/custom.js:13]: Yes.

**Review by chungthuang — APPROVED** [review]

**Pull request merged**